measures/2023/measures-data.json

```diff
@@ -26708,7 +26708,7 @@
         "name": "screened"
       },
       {
-        "description": "Percentage of patients aged 18 years and older who were identified as a tobacco user during the measurement period who received tobacco cessation intervention during the measurement period or in the six months prior to the measurement period",
+        "description": "Patients aged 18 years and older  who were identified as a tobacco user during the measurement period and who received tobacco cessationintervention during the measurement period or in the six months prior to the measurement period",
```

Comment:
I'm guessing there should be a space between cessation and intervention?

I also believe there is an extra space between older and who.

Reply:
you are correct - pls update